hera_cal/abscal.py

```diff
@@ -709,7 +755,13 @@ def global_phase_slope_logcal(model, data, antpos, wgts=None, refant=None, verbo
     fit : dictionary containing frequency-indpendent phase slope, e.g. Phi_ns_x
           for each position component and polarization [radians / meter].
     """
-    echo("...configuring linsolve data for global_phase_slope_logcal", verbose=verbose)
+    # check solver and edgecut
+    if solver == 'linfit':
+        echo("...configuring linsolve data for global_phase_slope_logcal", verbose=verbose)
+    elif solver == 'dft':
+        echo("...finding global phase slopes using the DFT method", verbose=verbose)
+    else:
+        raise ValueError("Unrecognized solver {}. Must be either 'linfit' or 'dft'.".format(solver))
```

Comment:
Would prefer an "assert solver in GLOBAL_LIST_OF_SOLVERS" and an echo that is not solver-dependent

Reply:
Done.